mobile-workspace/latest/using/index.md

```diff
@@ -47,6 +47,13 @@ You can upload single or multiple files within the Mobile Workspace.
 
 This will open your file repository on your mobile device (for example, Files for iPhone users) where you can select files to upload. This will upload files to the Personal files area. If you want to upload to a specific folder, create a new folder first and upload from inside that folder.
 
+To upload photos, you can use the following options:
+
+ | Option | Description |
+ | -------- | ----------- |
+ | **Upload photos and videos** | Photos are uploaded without EXIF metadata. |

```

Comment:
Photos are automatically uploaded without EXIF metadata according to the device security system.